README.md

```diff
@@ -37,7 +37,7 @@ Plasma chain itself has very simple structure with assets being undividable and
 
 Block in Plasma network has a structure of Header:[BlockNumber, NumberOfTransactions, ParentHash, MerkleTreeRoot, PlasmaOperatorSignature], where ParentHash references previous block (by number) and MerkleTreeRoot is root hash of a Merkle tree NumberOfTransactions transacitons in this Plasma block and array of transactions.
 
-Header is submitted by Plasma network operator to the smart-contract on Ethereum chain. Blocks can only be sent one by one, with sequence numbering is enforced by contract. Any user of Ethereum network can deposit ETH to contract that will trigger and event and will allow Plasma network operator to make a funding transaction in Plasma chain. Than users can freely transact in Plasma chain, with headers pushed to parent contract in Ethereum.
+Header is submitted by Plasma network operator to the smart-contract on Ethereum chain. Blocks can only be sent one by one, with sequence numbering is enforced by contract. Any user of Ethereum network can deposit ETH to contract that will trigger and event and will allow Plasma network operator to make a funding transaction in Plasma chain. Then users can freely transact in Plasma chain, with headers pushed to parent contract in Ethereum.
 
 When user wants to settle one of his transactions to the main network, he starts a withdraw on Ethereum network by providing reference to the transaction (in a form of BlockNumber, TxNumberInBlock, OutputNumberInTX), full transaction and Merkle proof that this transaction was indeed included in that block. Parent contract checks a proof versus submittet root hash for this block and if it passed starts withdraw process. After 24 hours it can be finalized. There is a special kind of transaction in Plasma netword that can speedup a process by effectively burning the input (sending it to 0x0). If this block is not published by operator, withdraw can go as normal.
```

Comment:
Perhaps include **submittet > submitted** on 3rd to last line and **netword > network** on 2nd to last line. Didn't want to to submit another PR for something so similar but just thought I would add these suggestions.
This is a great project!
Cheers.